Core/include/Acts/Material/VolumeMaterialMapper.hpp

```diff
@@ -166,6 +174,16 @@ class VolumeMaterialMapper {
   void collectMaterialSurfaces(State& /*mState*/,
                                const TrackingVolume& tVolume) const;
 
+  /// @brief Create extra material point for the mapping
```

Comment:
```suggestion
  /// Create extra material point for the mapping.
```
An explicit `@brief` is not required.

Reply:
Should I remove all the @brief from the volume mapping then ?

Reply:
In my view yes, but not in this PR. I just mentioned this here because it was changed.